components/x-gift-article/src/Message.jsx

```diff
@@ -12,8 +12,41 @@ export default ({
 	monthlyAllowance,
 	nextRenewalDateText,
 	redemptionLimit,
-	invalidResponseFromApi
+	invalidResponseFromApi,
+	isArticleSharingUxUpdates
 }) => {
+	if (isArticleSharingUxUpdates) {
```

Comment:
Not sure but I think currently this will always be true if the 'isArticleSharingUxUpdates ' flag exists and you are using a A/B flag. Unless I've missed something you should expect 'control' or 'variant' here or in the case of no flag 'false'. So the only condition this would be 'false' if the flag is not available.